app/views/content/non-uk-teachers/why-train-to-teach-transcript.md

```diff
@@ -59,8 +59,6 @@ The value of gaining a transferable qualification through training to teach in E
 
 ### Help for non-UK trainees 
 
-If you are interested in training to teach languages or physics, you could be eligible for up to £39,000 in a bursary or scholarship and international relocation payment. 
+If you are interested in training to teach languages or physics, you could be eligible for a bursary or scholarship worth up to £30,000. 
```

Comment:
@laurahermionetennant92 you're

Reply:
You will also need to add a redirect if you are removing the transcript page (suggest redirecting to train as a student page)